docs/source/checks/vision/methodology/plot_simple_feature_contribution.py

```diff
@@ -180,7 +174,7 @@ def mnist_batch_to_images_with_bias(batch):
 
 #%%
 
-check = SimpleFeatureContribution()
+check = SimpleFeatureContribution(per_class=False)
```

Comment:
But... you just reverted the doc to use the per_class=False. But we do want to show the per_class=True feature. We actually want to show both.

Reply:
it has both

Reply:
mnist is per class

Reply:
:+1: